api/app/googleflood.py

```diff
@@ -0,0 +1,184 @@
+"""Get data from Google Floods API"""
+
+import logging
+from concurrent.futures import ThreadPoolExecutor, as_completed
+from datetime import datetime, timezone
+from os import getenv
+from urllib.parse import urlencode
+
+from fastapi import HTTPException
+
+from .utils import make_request_with_retries
+
+logger = logging.getLogger(__name__)
+
+GOOGLE_FLOODS_API_KEY = getenv("GOOGLE_FLOODS_API_KEY", "")
+if GOOGLE_FLOODS_API_KEY == "":
+    logger.warning("Missing backend parameter: GOOGLE_FLOODS_API_KEY")
+
+
+def format_gauge_to_geojson(data):
+    """Format Gauge data to GeoJSON"""
+    geojson = {
+        "type": "Feature",
+        "geometry": {
+            "type": "Point",
+            "coordinates": [
+                data["gaugeLocation"]["longitude"],
+                data["gaugeLocation"]["latitude"],
+            ],
+        },
+        "properties": {
+            "gaugeId": data["gaugeId"],
```

Comment:
How confident are we that all the keys will be present? Should ze use a getter with a default None to be safe?
Or make sure that we have proper error handling for missing keys.

Reply:
We're pretty confident that they'll be present but there's no downside to the default none to be safe. I've updated to add a getter for non-essential values and error handling for missing essential values